typedoc.json

```diff
@@ -3,6 +3,20 @@
         "packages/*"
     ],
     "exclude": [
+        "packages/cjs-module-analyzer",
+        "packages/compartment-mapper",
+        "packages/daemon",
+        "packages/env-options",
+        "packages/errors",
```

Comment:
um

Reply:
https://endojs.github.io/endo/modules/_endo_errors.html is blank.

I took that to mean that there wasn't anything anybody wanted to say to a larger audience.

Reply:
Sigh. We do need to explain this API. Searching for `assert` in `*.md`, a lot of its mentions are in https://github.com/endojs/endo/blob/master/packages/ses/src/error/README.md and https://github.com/endojs/endo/blob/master/packages/ses/docs/lockdown.md . 

There are also the doc-comments in https://github.com/endojs/endo/blob/master/packages/ses/src/error/types.js and https://github.com/endojs/endo/blob/master/packages/ses/src/error/assert.js . Not sure how useful these are as starting points though.

Reply:
On env-options above, I make the point

> Certainly, we don't need to expose it immediately, as none of their use of anything else hinges on their ability to use the env-options API. But I think it is a nice development convenience.

I must reluctantly observe that this is true for `errors` too. So if you want to exclude both at this stage, the PR would still LGTM. Your choice.

Reply:
The docs for the ses `assert` are... not straightforward to deal with.

https://github.com/endojs/endo/blob/master/packages/ses/src/error/types.js has all sorts of detailed doc on stuff like `DetailsTag`. But ... it's all clobbered by https://github.com/endojs/endo/blob/master/packages/ses/types.d.ts

Reply:
That clobbering is interesting. Does that mean that, in general, when both files exist none of the doc-comments in the `types.js` file will make it to the typedoc?

Reply:
evidently so.

escalated / reified:

 - #2183

Reply:
took errors out too